scripts/globals/bluemagic.lua

```diff
@@ -437,7 +437,12 @@ xi.spells.blue.applySpellDamage = function(caster, target, spell, dmg, params, t
             return dmg
         end
 
+        -- Check if the mob has a damage cap
+        dmg = target:checkDamageCap(dmg)
```

Comment:
Does this _have_ to be before oneforall? Otherwise you could combine the two cap calls and move just outside of the next end block

Reply:
Realistically, the target wont ever have both one for all AND a damage cap.

However, I think damage cap should come/be applied AFTER all damage calculations, including phalanx, one for all AND stoneskin

Reply:
I can move it to after, I wasn't exactly sure *where* to put it but thinking about it again I think that makes the most sense.

Reply:
I don't think it matters, but afaik is untestable. The difference between before and after is one makes stoneskin and magic stoneskin godlike and the other makes it trivial 

Unless contagion transfer can move buffs into mobs that is...

Reply:
As far as I know the handful of things that actually have a damage cap doesn't have stoneskin so we can't really test it

Reply:
We may want to defer to "what's worse for the player"

which would be damage cap first, because that means even a weak stoneskin could be impenetrable without dispel, or require different strats to eat through (such as DW + enspells) if they are dispel immune

Reply:
After talking to winter about this I think moving this *BEFORE* stoneskin is the correct call. After looking at the crab bcnm that can potentially use stoneskin (I have asked for a retail capture but this may take a while as its not confirmed it can even cast it) it would get knocked off in one hit if this check was after.